### DISCUSSION TITLE
Add commenter or involves qualifier to shortcuts

### DISCUSSION THREAD

**Discussion by depascalis** [community/community#15334] [Mobile]
My main use case in mobile GitHub is to keep track of issues I'm somewhat involved in. In most cases I've made a comment on the issue or I'm just subscribing to it. I haven't found a way to easily follow these issues with the mobile app, but the search API has search qualifiers that would allow this.

**Solution**
Please add the available `commenter` or the `involves` search qualifier as filter options when creating custom shortcuts.

**Comment by stevepopovich** [comment]
Hey there! Thanks for the feedback. I can see how this would be useful. However, have you tried using the notification system? It should send you notifications when things happen with issues you are involved with.

**Reply by depascalis** [reply]
Hey @stevepopovich! Yes notifications are working fine, but there are instances when I have forgotten to subscribe, or somehow messed up the subscription, or just unsubscribed. The `involves` qualifier would guarantee that I can review everything I'm involved with.

**Reply by depascalis** [reply]
I now see that this request maybe should be addressed at notifications in general since the same issue exists on https://github.com/notifications. I have userscripts to get around the issue for my desktop browser, so it doesn't bother me as much there, but we don't have that control over the mobile app.

**Reply by stevepopovich** [reply]
Right. Technically, we share the same query capabilities for searching for things like this. So it would be a github.com feature. But do you want this feature for search or the option to see this as a shortcut in the mobile?

**Reply by depascalis** [reply]
I'll start by apologizing for the messy thread I'm creating now. 

I've been confusing myself. The `reason:participating` query seems to be doing what the topic title asks for, I can't remember why I've been needing `commenter` and `involves` at the moment, but I will certainly encounter the situation again.

I initially believed that the Search API was used everywhere, even for Notifications. So the limitations I've been experiencing is because they are separate, and to tackle that is just too much of a request.

It looks like github.com custom notification filters is the same as mobile Shortcuts.
So, what I'm really missing is to be able to create Shortcuts using the whole search API, but I understand that this is out of the scope for Shortcuts since shortcuts actually are notification filters.

**Solution**
1. Rename Shortcuts to what it actually is => custom notification filters
2. Repurpose Shortcuts and enable users to create shortcuts using the Search API (this would be great for github.com too)

Again, sorry for the mess 😂

**Reply by stevepopovich** [reply]
No, it's a tough problem to solve! I appreciate you working through it with me. 

1. So I disagree that Shortcuts are custom notifications. Shortcuts really are shortcuts to certain screens you are looking for. Like you can create a shortcut to a Repo screen. That has nothing to do with notifications. So I think selling this rename to my design team will be tough 😅 
2. This is a fair suggestion. It would be cool to have custom search queries saved so they can be quickly referenced.

Does this make sense?

**Reply by depascalis** [reply]
Thanks for the clarification on .1, I was just assuming and didn't know its full scope.

Makes absolutely sense, thank you!

**Comment by AMDphreak** [comment]
Still a great ask ? +1, especially for shortcuts and filter UI parity.

Search already supports `commenter:` / related qualifiers, but the Issues/PRs filter dropdowns advertise **Mentions** while leaving **Commenter** (and **Involves**) out. That is the gap that keeps biting people who want a durable, shareable filtered URL for hiring or collaboration.

I opened an updated request focused on that UI omission: https://github.com/orgs/community/discussions/202870

Would be excellent to see Commenter/Involves in both filters and shortcuts.

**Comment by arao72192-dotcom** [comment]
Gi
